src/app/[locale]/(app)/(nav)/(time-selector)/stats/_components/SendingTimes.tsx

```diff
@@ -32,11 +32,11 @@ function Stats() {
   const data = [
     {
       value: formatNumber(avgMessagesSentPerDay),
-      label: "average messages sent per day",
+      label: "overview.averageMessage",
```

Comment:
For all of the labels, call `t` after importing it and adding the "use client" directive